README.md

```diff
@@ -38,27 +42,31 @@ To install this package into a workspace already containing SOSS, just clone thi
 
 5. Source a colcon environment in which ROS2 has been built (soss-ros2 uses rclcpp package).
     ```
-    source path/to/ros2/ws/install/local_setup.bash
+    $ source path/to/ros2/ws/install/local_setup.bash
     ```
 
 6. In the workspace folder, execute colcon:
     ```
-    colcon build --packages-up-to soss-dds soss-ros2
+    $ colcon build --packages-up-to soss-dds soss-ros2
     ```
+
 7. Source the current environment
     ```
-    source install/local_setup.bash
+    $ source install/local_setup.bash
     ```
 
 ## Usage
 
 This system handle is mainly used to connect any system with the DDS protocol.
+Exists two main usages of this *SOSS DDS plugin`:
```

Comment:
Instead of "exists" I'd say "There are". And instead of "usages", for me "use cases" sounds better, but maybe here it also sounds like something not related with the system handle directly, as we already talked the other day.

Reply:
I see the "use case" as a real example using a technology. For example: Use case using tcp tunnel to connect a ros2 with a fiware server in another network. You have several usages of a thing, and you can have different cases for an usage. Even more exagerate, my use case could be connect ros2 to fiware at different clouds, and to address the problem I use the TCP tunnel mode that DDS offers me. 
Any way, I changed it to: `There are two communication modes of *SOSS DDS plugin:` that maybe is more concise. Also, I modified the lines below.